semanticcpg/src/main/scala/io/shiftleft/semanticcpg/sarif/SarifConfig.scala

```diff
@@ -0,0 +1,26 @@
+package io.shiftleft.semanticcpg.sarif
+
+import io.shiftleft.semanticcpg.sarif.SarifConfig.SarifVersion
+import io.shiftleft.semanticcpg.sarif.v2_1_0.JoernScanResultToSarifConverter
+import org.json4s.Serializer
+
+import java.net.URI
+
+case class SarifConfig(
```

Comment:
Do we really need a separate `SarifConfig` classs if we already have a "Sarif converter" which could easily contain the values from the config class in addition to the implementation it currently has.

Reply:
My idea is to be consistent with the data-flow engine style of having a single constant config that the queries pick up on in the REPL. The configuration itself rarely changes, and is only necessary to configure once for compulsory tool-related meta-data. 

In the scope of a REPL, the configuration will generally not change, but one may call `toSarifJson` multiple times for possibly different scans.